DOCS/man/options.rst

```diff
@@ -5555,15 +5555,15 @@ them.
         Disable any dithering done by mpv.
     auto
         Automatic selection.
-        On ``vo=gpu``: if output bit depth cannot be detected, 8 bpc is assumed.
-        On ``vo=gpu-next``: with ``gpu-api=d3d11``, real on-the-wire bpc is used.
-        For other ``gpu-api``, 8 bpc is used for SDR content.
+        On ``--vo=gpu``: detected depth or 8 bpc otherwise
+        On ``--vo=gpu-next``: detected depth or 8 bpc (SDR content) or no
```

Comment:
Not `(SDR content)`. It is `target`. If SDR content with HDR `target-trc` would not use 8 bpc.

Reply:
It should be fine no? We use the same language for --target-trc and saying SDR content should get the point across better than "for SDR target transfer"